workspace.json

```diff
@@ -0,0 +1,113 @@
+{
+  "version": 2,
+  "cli": {
+    "defaultCollection": "@nrwl/react"
+  },
+  "generators": {
+    "@nrwl/react": {
+      "application": {
+        "babel": true
+      }
+    }
+  },
+  "projects": {
+    "spark": {
+      "root": "libs/spark",
+      "sourceRoot": "libs/spark/src",
+      "projectType": "library",
+      "targets": {
+        "build": {
+          "executor": "@nrwl/web:package",
+          "outputs": ["{options.outputPath}"],
+          "options": {
+            "outputPath": "dist/libs/spark",
+            "tsConfig": "libs/spark/tsconfig.lib.json",
+            "project": "libs/spark/package.json",
+            "entryFile": "libs/spark/src/index.ts",
+            "external": ["react", "react-dom", "react-is", "styled-components"],
```

Comment:
The `external` property lists dependencies that will not be bundled with the library. We could add `@material-ui` to that list if we expect that consumers of `@prenda/spark` will already be using it.

Reply:
In my main review comment, i pasted the outputted package.json from the distribution build script. It includes `@material-ui` as a peer dependency so i think somewhere along the way, it's already getting marked as equivalent to external.

I think it would be good for us to add it to `external` there as well just so that's clarified from our POV.

Additionally, this goes towards solving an issue some devs have come to me with, where spark's theme object is not accessible through context to any context-consumer function they use. It's because spark's installed MUI pkg is not their project's installed MUI pkg and so the context instance is different and their imports like `import { withStyles } from '@material-ui/core'` end up disconnected. There's more than one way to solve that problem, but this is probably the best way.

Reply:
I took your suggestion, and that change is in the latest commit.